proto/ignition/msgs/geometry.proto

```diff
@@ -51,7 +51,9 @@ message Geometry
     LINE_STRIP   = 8;
     POLYLINE     = 9;
     CONE         = 10;
-    EMPTY        = 11;
```

Comment:
Is there a reason for moving `EMPTY` to the bottom of the list? I think this will break backwards compatibility.

Reply:
Nope. I just assumed it needed to stay at the end. I will append CONE.

Reply:
The order is corrected.

Reply:
Looks good, thanks!